### ISSUE TITLE
Repair Safegraph changeset from #309

### ISSUE THREAD

**Issue by krivard** [cmu-delphi/covidcast-indicators#337]
What was done:

* #309 was merged to `deploy-safegraph`
* Those changes were propagated to `main` to unblock downstream work
* #336 was merged to `deploy-safegraph` to back out those changes, to attempt to resolve a failing Safegraph pipeline run

What should happen next:

- [ ] Verify that Safegraph now runs to completion (ETA Wednesday morning)
- [ ] Fix #309
- [ ] Reapply to `deploy-safegraph`
- [ ] Propagate to `main`

Relevant log segment from pipeline run:
```
Starting step [id=9850|step_id=69|name=Run Indicator: SafeGraph]
   Running cmd...
Purging ./receiving...
Running the indicator...
Completed 256.0 KiB/98.9 MiB (1.2 MiB/s) with 1 file(s) remaining
Completed 512.0 KiB/98.9 MiB (2.3 MiB/s) with 1 file(s) remaining
[...]
Completed 98.1 MiB/98.9 MiB (169.6 MiB/s) with 1 file(s) remaining
Completed 98.4 MiB/98.9 MiB (169.8 MiB/s) with 1 file(s) remaining
Completed 98.6 MiB/98.9 MiB (170.0 MiB/s) with 1 file(s) remaining
Completed 98.9 MiB/98.9 MiB (170.3 MiB/s) with 1 file(s) remaining
download: s3://sg-c19-response/social-distancing/v2/2020/10/16/2020-10-16-social-distancing.csv.gz to ../../../../common/safegraph/social-distancing/2020/10/16/2020-10-16-social-distancing.csv.gz
Traceback (most recent call last):
  File "/home/indicators/.pyenv/versions/3.8.2/lib/python3.8/runpy.py", line 193, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/home/indicators/.pyenv/versions/3.8.2/lib/python3.8/runpy.py", line 86, in _run_code
    exec(code, run_globals)
  File "/home/indicators/runtime/safegraph/delphi_safegraph/__main__.py", line 11, in <module>
    run_module()  # pragma: no cover
  File "/home/indicators/runtime/safegraph/delphi_safegraph/run.py", line 64, in run_module
    pool.map(process_file, files)
  File "/home/indicators/.pyenv/versions/3.8.2/lib/python3.8/multiprocessing/pool.py", line 364, in map
    return self._map_async(func, iterable, mapstar, chunksize).get()
  File "/home/indicators/.pyenv/versions/3.8.2/lib/python3.8/multiprocessing/pool.py", line 768, in get
    raise self._value
  File "/home/indicators/.pyenv/versions/3.8.2/lib/python3.8/multiprocessing/pool.py", line 537, in _handle_tasks
    put(task)
  File "/home/indicators/.pyenv/versions/3.8.2/lib/python3.8/multiprocessing/connection.py", line 206, in send
    self._send_bytes(_ForkingPickler.dumps(obj))
  File "/home/indicators/.pyenv/versions/3.8.2/lib/python3.8/multiprocessing/reduction.py", line 51, in dumps
    cls(buf, protocol).dump(obj)
AttributeError: Can't pickle local object 'run_module.<locals>.process_file'
Copying files to the ingestion directory...
cp: cannot stat ‘receiving/202009*’: No such file or directory
   Fail [cmd=sudo -u indicators -s bash -c "cd /home/indicators/runtime/safegraph && bash run-safegraph.sh"]
```

**Comment by sgsmob** [comment]
The `AttributeError: Can't pickle local object 'run_module.<locals>.process_file'` message likely corresponds to pickling a local function, which apparently [does not generally work](https://stackoverflow.com/questions/52265120/python-multiprocessing-pool-map-attributeerror-cant-pickle-local-object).  I will try to rewrite to use `itertools.partial`.

**Comment by sgsmob** [comment]
I also inadvertently changed the output file format to write the date as YYYY-MM-DD rather than YYYMMDD, which caused the `cp: cannot stat ‘receiving/202009*’: No such file or directory` error.  Fixed and re-running indicator to verify the behavior is correct.

**Comment by sgsmob** [comment]
Can this issue be closed now that #345 is in?
